ambry-api/src/main/java/com/github/ambry/config/CloudConfig.java

```diff
@@ -351,6 +364,34 @@
   @Default(DEFAULT_VCR_HELIX_STATE_MODEL_FACTORY_CLASS)
   public final String vcrHelixStateModelFactoryClass;
 
+  /**
+   * The vcr node responsible for partition 1 is the vcr helix updater.
+   */
+  @Config(VCR_HELIX_UPDATER_PARTITION_ID)
+  @Default("-1")
```

Comment:
Since this is a string, could the default be empty string "" to disable the feature?

Reply:
yes. Empty string looks good.